akka-cluster/build.sbt

```diff
@@ -9,4 +9,9 @@ Dependencies.cluster
 // disable parallel tests
 parallelExecution in Test := false
 
+// pass on the artery enable/disable flag for the tests
+jvmOptions in MultiJvm ++=
+  (if (sys.props.get("akka.cluster.test.use-artery").exists(_ == "true")) Seq("-Dakka.cluster.test.use-artery=true")
```

Comment:
It passes the props of the sbt JVM to all the multinode JVMs automagically?? O.o